.travis.yml

```diff
@@ -1,43 +1,36 @@
+    - secure: atEtVos72LKeP2RkvJe3Kv775yK8YkbzdzVBv03KSWkHNqxMLFxxY03J1rJpWWyW+jEGZUjYGKtH9WIWMXqJOHkIqVkbxtvHerDBjL51sWndU6h0YQgySwXilH6nY6oWXL4ul+N/nYUiEF1sJwHDr04tUMJws7707gbZsNdN4EWELfcfpSm7GyhhPKRMNoIj1LPHrEx690EnpypGAUbM4zx55wXP8hfbXxULF+dWg43/GQVgBwi0xVxhfS17c3hMfa4Ohi8j7EacNuBcTNQLIxUcpzlNxGDpLTUfkHWNZ4soBJn6ZMyNO79Far+CsbKhqe0K08Y9wIgrz741ORixAWj7x/10Al9Q7qQ3i7R0m2ZtIbhVmFc9lYAs3U/HQI08OIOOo3uaOvwQxJDFwCqRKL1ckRfZ5Wz8ax2hT75r9w2hiJQ32RNJHswQGPEhwe0jV5Q5krumdAnI7DdRt8JbO8neugeFa4gzVPc9P1tRzZ2n755ckaGRoSu18YqxX9wUL+dIzoY9vh6r8Bkv1eaNi4kTeWl8Cy1l0ObdcpzYcsCMngDdVpmsFyr3fd3i9+dRkPQLXEaK38T6PakLA+y31I48w0lBhBCNKfi9Z1fRl92EXxlYuTqb4EdDlfxHV6GfngZVBkeNOdA5/P882SmYiQ88KF8Wlq/SD6NGW1Nuz6I=
 
-bundler_args: --binstubs=./bundler_stubs
+bundler_args: "--binstubs=./bundler_stubs"
 
 before_script:
-  - npm install -g jshint coffeelint htmlhint csslint # overcommit packages
-  - cp spec/dummy/config/database.travis.yml spec/dummy/config/database.yml
-  - RAILS_ENV=test bundle exec rake db:create
+- npm install -g jshint coffeelint htmlhint csslint
+- cp spec/dummy/config/database.travis.yml spec/dummy/config/database.yml
+- RAILS_ENV=test bundle exec rake db:create
 
 script:
-  - bundle exec rake --trace before_commit:run_without_checks
-  - htmlhint -c ./.htmlhintrc ./app/**/*.html.erb
-  - git config --global user.email test@example.com # overcommit Author
-  - git config --global user.name "Test Example" # overcommit Author
-  - BUNDLE_GEMFILE=.overcommit_gems.rb bundle exec overcommit --run
-  - RAILS_ENV=test bundle exec rake db:migrate --trace
-  - bundle exec rspec
+- bundle exec rake --trace before_commit:run_without_checks
+- htmlhint -c ./.htmlhintrc ./app/**/*.html.erb
+- git config --global user.email test@example.com
+- git config --global user.name "Test Example"
+- BUNDLE_GEMFILE=.overcommit_gems.rb bundle exec overcommit --run
+- RAILS_ENV=test bundle exec rake db:migrate --trace
+- bundle exec rspec
 
-# Have this option to stop travis-ci building twice. Currently we have travis set to build both
-# PR's and pushes. However this means when we push to a PR we have to wait for Travis to finish
-# 2 builds. If we unticked 'pushes' when the PR was finally merged that would not be built. The
-# brute force approach would be to untick build PR's and just build all pushes. We instead have
-# gone with the approach outlined here http://stackoverflow.com/a/31882307
 branches:
   only:
-    - master
-    - develop
+  - master
+  - develop
```

Comment:
interestingly the travis cmd line tool did this when generating new secure env vars